Loop/Managers/DeviceDataManager.swift

```diff
@@ -1210,6 +1211,67 @@ extension DeviceDataManager {
                 case .cancelTemporaryOverride:
                     log.default("Canceling temporary override from remote command")
                     loopManager.settings.scheduleOverride = nil
+                case .bolusEntry(let bolusAmount):
+                    log.default("Enacting remote bolus entry: %{public}@", String(describing: bolusAmount))
+                    
```

Comment:
Add explicit guards for the various error conditions on a remote bolus. In there is an error, that will be reported back as a user local notification.